client/src/components/ReviewContent/ReviewContent.css

```diff
@@ -15,12 +15,12 @@
 .inline_block_smaller {
   flex: 30%;
   padding-right: 20px;
-  text-align: right;
   white-space: normal;
   font-size: 14px;
   font-style: normal;
   text-transform: none;
   line-height: 20px;
+  padding-top:60px;
```

Comment:
I cannot make any comment on css. It would be nice if @jonfu86 can make some comments.